### ISSUE TITLE
[Bug] 使用自定义字体后出现空白占位

### ISSUE THREAD

**Issue by vision-yip** [apache/echarts#18991]
### Version

5.3.2

### Link to Minimal Reproduction

https://codepen.io/nuknjgod-the-reactor/pen/zYMgyxJ

### Steps to Reproduce

1. 创建chart, 并使用rich
2. 加载自定义字体
3. 加载完成自定义字体后重新渲染chart

### Current Behavior

rich标签value前后有空占位, 如使用注释代码则无前后空占位;(其实是数据发生变化后会正常, 但是使用原数据会有空占位)

### Expected Behavior

无空占位

### Environment

```markdown
- OS:macOS
- Browser:Chrome 96.0.4664.55
- Framework: vue@2.6.4
```

**Comment by helgasoft** [comment]
> If you use the comment code, there will be no spaces

Indeed, you need to use this code - _loadFont_ happens after showing the initial chart.
The initial chart ignores ```fontFamily:"font5fe5937d581913001efadb24"``` as unknown, and uses a default font since the custom has not been loaded yet.
 Try it by commenting out ```fontFamily``` and you'll see the same (default) font.

**Comment by vision-yip** [comment]
> > If you use the comment code, there will be no spaces
> 
> Indeed, you need to use this code - _loadFont_ happens after showing the initial chart. The initial chart ignores `fontFamily:"font5fe5937d581913001efadb24"` as unknown, and uses a default font since the custom has not been loaded yet. Try it by commenting out `fontFamily` and you'll see the same (default) font.

Yes, you're right. 
But I hope to use the source data to re render without any spaces, Because I use other data to re render without spaces

**Comment by vision-yip** [comment]
> I don't think there is any problem and I agree with what @helgasoft said. You need to make sure the custom fonts have been loaded before rendering the chart if you don't want a refresh step.

The product requirement is to render the chart first, as loading the custom font may take some time, which is not good for the user experience😞. After loading the custom font, use the custom font to display it;
